wsdiscovery/message.py

```diff
@@ -35,7 +35,12 @@ def parseSOAPMessage(data, ipAddr):
         #print('Fault received from %s %s:' % (ipAddr, data), file=sys.stderr)
         return None
 
-    soapAction = dom.getElementsByTagNameNS(NS_ADDRESSING, "Action")[0].firstChild.data.strip()
+    actions = dom.getElementsByTagNameNS(NS_ADDRESSING, "Action")
+    if len(actions) == 0:
+        #print('No action received from %s %s' % (ipAddr, data), file=sys.stderr)
```

Comment:
Let's make this print `logger.warning`
would help somebody to understand what is going on

Reply:
done, adjusted the other prints as well